packages/iov-bcp/src/transactions.spec.ts

```diff
@@ -1,6 +1,152 @@
-import { isBlockHeightTimeout, isTimestampTimeout } from "./transactions";
+import { Encoding } from "@iov/encoding";
+
+import {
+  Algorithm,
+  ChainId,
+  Identity,
+  isBlockHeightTimeout,
+  isIdentity,
+  isNonNullObject,
+  isPubkeyBundle,
+  isTimestampTimeout,
+  isUint8Array,
+  PubkeyBundle,
+  PubkeyBytes,
+} from "./transactions";
+
+const { fromHex } = Encoding;
 
 describe("transactions", () => {
+  describe("isNonNullObject", () => {
+    it("returns true for objects", () => {
```

Comment:
"including arrays" to help with clarity here?

Reply:
Hah! I bend the tests to the behavior of JavaScript. My original expectation was `expect(isNonNullObject([])).toEqual(false);`.

Looking at the definition of the `object` type, an Array is an object:

> object is a type that represents the non-primitive type, i.e. anything that is not number, string, boolean, symbol, null, or undefined.

https://www.typescriptlang.org/docs/handbook/basic-types.html#object

So 👍 for making that explicit